Mathlib/Topology/Algebra/ConstMulAction.lean

```diff
@@ -173,7 +173,8 @@ theorem smul_closure_orbit_subset (c : M) (x : α) :
     c • closure (MulAction.orbit M x) ⊆ closure (MulAction.orbit M x) :=
   (smul_closure_subset c _).trans <| closure_mono <| MulAction.smul_orbit_subset _ _
 
-theorem isClosed_setOf_map_smul {N : Type*} [Monoid N] (α β) [MulAction M α] [MulAction N β]
+omit [Monoid M] in
+theorem isClosed_setOf_map_smul {N : Type*} (α β) [SMul M α] [SMul N β]
```

Comment:
AFAIR, we have a section for `SMul` in this file. Could you please move this lemma there?